src/app/external-services/views/ExternalServiceListView.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <DataSource
+    v-slot="{ data: me }: MeSource"
+    src="/me"
+  >
+    <RouteView
+      v-if="me"
+      v-slot="{ can, route, t }"
+      name="external-service-list-view"
+      :params="{
+        page: 1,
+        size: me.pageSize,
+        mesh: '',
+      }"
+    >
+      <DataSource
+        v-slot="{data, error}: ExternalServiceCollectionSource"
+        :src="`/meshes/${route.params.mesh}/external-services?page=${route.params.page}&size=${route.params.size}`"
+      >
+        <AppView>
+          <template #title>
+            <h2>
+              <RouteTitle
+                :title="t('external-services.routes.items.title')"
+              />
+            </h2>
+          </template>
+
+          <template
+            v-if="can('use gateways ui')"
+            #actions
+          >
+            <LinkBox>
+              <RouterLink
+                :to="{
+                  name: 'service-list-view',
+                  params: {
+                    mesh: route.params.mesh,
+                  },
+                }"
+              >
+                Internal
+              </RouterLink>
+
+              <RouterLink
+                :to="{
+                  name: 'external-service-list-view',
+                  params: {
+                    mesh: route.params.mesh,
+                  },
+                }"
+              >
+                <b>External</b>
+              </RouterLink>
+            </LinkBox>
```

Comment:
I haven’t looked at the design yet so I’m not entirely sure what we’ll need. In any case, this is unfinished and I’ll pass over this again once we have them. During that step, it might well be that I’m swapping this out again.

In short, there will be a PR adjusting the changes here (and the future ones for gateways) to the design.

Reply:
For reference: https://kongponents.konghq.com/components/segmented-control.html

It looks like KSegmentedControl always wraps its elements into a button element with click handler. This would be problematic for us since we’d be using links inside the elements.

Reply:
Doh, yeah not ideal at all. We should consider bring this up with the kongponents folks I think, feels very much like the KTabs component not being able to work with links, we can chat about that later. Will resolve this for the moment.